projects/museumpassmusees/reference/partner-api.json

```diff
@@ -430,9 +430,188 @@
           "Visits"
         ]
       }
+    },
+    "/external-cards/{type}/{identification}": {
+      "get": {
+        "summary": "Look up external card",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "visit": {
+                      "type": "object",
+                      "required": [
+                        "allowed"
+                      ],
+                      "description": "Visit information for the external card.",
+                      "properties": {
+                        "allowed": {
+                          "type": "boolean",
+                          "description": "Indicates whether a visit with the external card is allowed or not."
+                        },
+                        "reason": {
+                          "type": "string",
+                          "enum": [
+                            "NO_SOCIAL_STATUS"
+                          ],
+                          "description": "Reason why a visit is not allowed."
+                        }
+                      }
+                    }
+                  },
+                  "required": [
+                    "visit"
+                  ]
+                },
+                "examples": {}
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "Not Found. Possible error types:\n\n* https://api.publiq.be/probs/url/not-found",
+            "content": {
+              "application/problem+json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          },
+          "503": {
+            "description": "Service Unavailable. Possible error types:\n\n* https://api.publiq.be/probs/museumpass/external-service-unavailable\n\nThe `Retry-After` HTTP Header might include a hint when a retry of this request can be attempted.\n\n",
```

Comment:
> Possible error types:\n\n* https://api.publiq.be/probs/museumpass/external-service-unavailable\n\n

For self-explanatory status codes you MAY also use `about:blank` as the type according to RFC7807. For example 403, 404 or 500. The reason that we use a custom type for 4XX errors even when not needed is that we can make it link to documentation to provide some more info how to solve it. But for 5XX errors I'm not sure that that is necessary since a client can not solve it on their end. So for 5XX errors I think `about:blank` is fine as type, and then you don't need to document the error type(s) here either.